hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -163,6 +164,19 @@ module top_${top["name"]} #(
   // this avoids lint errors
   assign unused_irq_id = irq_id;
 
+  // Alert list
+  prim_pkg::alert_tx_t [alert_pkg::NAlerts-1:0]  alert_tx;
+  prim_pkg::alert_rx_t [alert_pkg::NAlerts-1:0]  alert_rx;
+  // Escalation outputs
+  prim_pkg::esc_tx_t [alert_pkg::N_ESC_SEV-1:0]  esc_tx;
+  prim_pkg::esc_rx_t [alert_pkg::N_ESC_SEV-1:0]  esc_rx;
+
+% if not top["alert"]:
+  // tie off if no alerts present in the system
+  assign alert_handler_alert_p = '0;
+  assign alert_handler_alert_n = '0;
```

Comment:
is a 00 tie off technically correct here?

Reply:
no, it should be '0 ad '1. will fix.